db/seeds.rb

```diff
@@ -147,12 +147,143 @@
                city: 'Washington', state: 'DC', zip: '20011'
 Clinic.create! name: 'Sample Clinic 2 - VA', street_address: '123 Fake Street',
                city: 'Arlington', state: 'DC', zip: '22204'
-                      
+
+# Add some patients with pledges some of whom have
+# fulfillments for reporting testing for fulfillments
+10.times do |i|
+  flag = i.even? ? true : false
+
+  patient = Patient.create!(
+    name: "Reporting Patient #{i}",
+    primary_phone: "263-0#{i}0-002#{rand(10)}",
+    initial_call_date: 3.days.ago,
+    urgent_flag: flag,
+    line: i.even? ? 'DC' : 'MD',
+    created_by: User.first,
+    clinic_name: "Sample Clinic #{rand(2)}",
+    appointment_date: 10.days.from_now,
+  )
+
+  pregnancy = patient.build_pregnancy(
+    last_menstrual_period_weeks: 1,
+    last_menstrual_period_days: 7,
+    created_by: User.first,
+    naf_pledge: 2000,
```

Comment:
Kind of nitpicky, but just so the seed data looks slightly more accurate, let's change these pledges to as follows:

NAF: 300
Procedure cost: 600
DCAF soft pledge: 200
patient contribution: 100